__tests__/fees.js

```diff
@@ -1,81 +1,101 @@
 describe('fees', () => {
-    beforeEach(() => {
-        jest.resetAllMocks()
-        jest.resetModules()
+    describe('real tests', () => {
+        const { get_fee_rate, get_min_next_block_fee_rate } = require('../fees')
+
+        it('should return positive fee', async () => {
+            expect.assertions(1)
+
+            const feeRate = await get_fee_rate()
+            expect(feeRate).toBeGreaterThan(0)
+        })
+
+        it('should return positive fee', async () => {
+            expect.assertions(1)
+
+            const feeRate = await get_min_next_block_fee_rate()
+            expect(feeRate).toBeGreaterThan(0)
+        })
     })
```

Comment:
this simulates real smoke test and ensures we receive actual fee, if there is regression it will return error and then ` expect.assertions(1)` will fail